Configuration/HLT/python/autoHLT.py

```diff
@@ -13,6 +13,7 @@
   'relval2022' : 'Fake2',
   'relval2023' : 'Fake2',
   'relval2024' : 'GRun',
```

Comment:
are we going to get 2025 relvals to run in IBs and PRs?

Reply:
IBs no, PRs I've added a single ttbar noPu.

Reply:
Meaning: I didn't add the in this PR but it would be doable.

Reply:
OK, I let you decide how to move forward. I assume at some point it will be necessary in any case and perhaps we could start doing it now.

Reply:
I've thought about this and it depends a bit also on how @cms-sw/alca-l2 foresee the evolution of 2024 GTs given this impact the release validations. Currently we run only 2024 wfs and, as is, this allows us to validate both the "latest" 2024 HLT (now `GRun`) and the latest symbolic GT. If we move the 2024 to `Fake` and the 2025 is promoted to `GRun` then I would still like to validate the latter moving the RelVals production to 2025 MC. But then the risk would be that we wouldn't validate the latest 2024 GTs (unless `auto:phase1_2025_*` are explicitly kept in sync or forced to be so in `autoCond.py`). So if AlCa tells me that for 2024 we are more or less done, ok then let's move everything to (2024-like) 2025 and we will move also the RelVal production to it.

Reply:
> But then the risk would be that we wouldn't validate the latest 2024 GTs 

mmh, what's the need to validate 2024 GTs in *this* cycle? I suppose they need to be backported anyway in 14.0.X where we don't plan to decommission the 2024 menu.

Reply:
Just because we run release validations mostly on this cycle and not on `14_0_X` (or `14_1_X`). I see it as a way of profiting of the regular scrutiny of validators to check also the new GTs. Similarly to what happened for HLT being indirectly checked on `14_1_X` during this whole period thanks to `@relval2024`.

Reply:
we're not going to re-run the 2024 HLT in any other release than 14.0.X in production. I don't see the reason to validate the 2024 HLT in this cycle. Of course "offline" validators can continue to validate offline reconstruction.

Reply:
This is clear, I wanted to understand if it's anyway useful to keep an eye on the 2024 GT (for the "offline" validations).

Reply:
If it's not, I'll move everything to 2025 and that's it.

Reply:
I've moved `@relval2024` to `Fake` and added a couple of 2025 wfs to IB tests.

Reply:
> This is clear, I wanted to understand if it's anyway useful to keep an eye on the 2024 GT (for the "offline" validations).

2024 pp GTs are supposed to be "completed" by now.
There is still some work in progress for the 2024 HI GTs, but they are validated via different workflows than the ones addressed here.